src/Agent/NewRelic/Agent/Extensions/Providers/Wrapper/AwsLambda/HandlerMethodWrapper.cs

```diff
@@ -77,9 +77,13 @@ public AfterWrappedMethodDelegate BeforeWrappedMethod(InstrumentedMethodCall ins
             EventTypes.TryGetValue(eventTypeName, out var eventType); // handle case where the name might not be in the eventType dictionary
 
             attributes.AddEventSourceAttribute("eventType", eventType ?? "Unknown"); // TODO: Is this correct?
-            attributes.Add("aws.RequestId", requestIdGetter(lambdaContext));
+            attributes.AddEventSourceAttribute("arn", "????"); // TODO: how to get this value? Spec says "ARN of the invocation source" 
```

Comment:
Should we hold off on adding this until we figure out how to get it?

Reply:
That's really more of a placeholder so I could PR the changes I've made so far. I'll continue refining things as we move along.

Reply:
That's fine, I just want to ensure that we don't forget to do that.